database/factories/ConsultationParticipantFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace EscolaLms\Consultations\Database\Factories;
+
+use EscolaLms\Auth\Models\User;
+use EscolaLms\Consultations\Enum\ConsultationParticipantStatusEnum;
+use EscolaLms\Consultations\Models\Consultation;
+use EscolaLms\Consultations\Models\ConsultationParticipant;
+use EscolaLms\Courses\Database\Factories\FakerMarkdownProvider\FakerProvider;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class ConsultationParticipantFactory extends Factory
+{
+    protected $model = ConsultationParticipant::class;
+
+    public function definition()
+    {
+        $this->faker->addProvider(new FakerProvider($this->faker));
```

Comment:
You are adding Markdown Faker provider and then not using it

Reply:
Removed this line